### PR TITLE
Add quest playmode to redux

### PR DESCRIPTION
**Pull request by toplenboren**
Hints are now being put into state client-side
TaskAttempts are now being put into state clien-side
Quest Playmode now uses redux only

### CODE REVIEW

File: src/components/QuestPlaymode/Templates/Tasks/Hint.jsx [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
+                    }
+                    onCancel={() => {
+                        console.log('closed');
+                    }}
+                    okText="Да"
+                    cancelText="Нет"
+                >
+                    <button className={'quest-task__button'}>
+                        <div className={'quest-task__typography__podskazka'}>
+                            <p>{'Подсказочку ❓'}</p>
```
Comment (tramakarov) [inline src/components/QuestPlaymode/Templates/Tasks/Hint.jsx:36]: Может здесь вставить `<QuestionOutlined />` вместо эмодзи?

File: src/components/QuestPlaymode/Templates/Tasks/QuestTaskGroup.jsx [hunk 2]
```diff
@@ -1,132 +1,159 @@
...
-        </Row>
-        <Divider/>
-      </React.Fragment>
-    )
-  }
+        console.log(taskData);
+        console.log(taskData.question);
+        return (
+            <React.Fragment>
+                <Row style={{ display: 'flex', color: '#000000' }}>
```
Comment (tramakarov) [inline src/components/QuestPlaymode/Templates/Tasks/QuestTaskGroup.jsx:89]: Hardcoded css detected :)

File: src/components/QuestPlaymode/Templates/Tasks/Tasks.css [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 }
-.quest-task__container{
+.quest-task__container {
     display: inline-flex;
     width: 33%;
     height: 22vh;
     padding-right: 10px;
 }
-.quest-task__container__podskazka{
+.quest-task__container__podskazka {
```
Comment (tramakarov) [inline src/components/QuestPlaymode/Templates/Tasks/Tasks.css:10]: quest-task__container__hint?

Comment (toplenboren) [reply src/components/QuestPlaymode/Templates/Tasks/Tasks.css:10]: :)

File: src/components/QuestPlaymode/Templates/Tasks/Hint.jsx (hunk 1 above)
Comment (toplenboren) [reply src/components/QuestPlaymode/Templates/Tasks/Hint.jsx]: Done, умерла верстка на мозилле. Поправлю later

**Pull request merged**